tests/foreman/ui/test_contentview.py

```diff
@@ -3524,6 +3524,38 @@ def test_positive_delete_composite_version(self):
                 composite_cv.name, version)
 
     @run_only_on('sat')
+    @tier2
+    def test_positive_delete_version_without_refresh(self):
+        """Publish content view few times in a row and then delete one version.
+
+        :id: 4c11eaaa-a9c0-4fbc-91f2-e8290dce09f5
+
+        :expectedresults: Version is deleted successfully and all other
+            versions are present on the page
+
+        :BZ: 1374134
+
+        :CaseLevel: Integration
+        """
+        org = entities.Organization().create()
+        cv = entities.ContentView(organization=org).create()
+        cvv_names = []
+        for i in range(5):
+            cv.publish()
+            cvv = cv.read().version[i].read()
```

Comment:
i'd drop counter `i`, accessed by the last element and probably checked len of versions just in case:
```python
        for _ in range(5):
            cv.publish()
            cv = cv.read()
            cvv = cv.version[-1].read()
            cvv_names.append('Version {0}'.format(cvv.version))
        self.assertEqual(len(cv.version), 5)
```
but your way isn't any worse, no action needed, it's just `for` loop with `i` counter reminds me C too much :)